hospital/src/main/java/by/emel/anton/Runner.java

```diff
@@ -1,26 +1,17 @@
 package by.emel.anton;
 
-import by.emel.anton.model.dao.implementation.FileDoctorDAO;
-import by.emel.anton.model.dao.implementation.FilePatientDAO;
-import by.emel.anton.model.dao.implementation.FileTherapyDAO;
-import by.emel.anton.model.dao.implementation.FileUserDAO;
-import by.emel.anton.model.dao.interfaces.DoctorDAO;
-import by.emel.anton.model.dao.interfaces.PatientDAO;
-import by.emel.anton.model.dao.interfaces.TherapyDAO;
-import by.emel.anton.model.dao.interfaces.UserDAO;
-import by.emel.anton.service.UserService;
-import by.emel.anton.service.UserServiceImpl;
 import by.emel.anton.terminalprog.TerminalProgram;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.ConfigurableApplicationContext;
 
+@SpringBootApplication
 public class Runner {
+
     public static void main(String[] args) {
 
-        UserDAO userDAO = new FileUserDAO();
-        PatientDAO patientDAO = new FilePatientDAO();
-        DoctorDAO doctorDAO = new FileDoctorDAO();
-        TherapyDAO therapyDAO = new FileTherapyDAO();
-        UserService userService = new UserServiceImpl(userDAO,patientDAO,doctorDAO,therapyDAO);
-        TerminalProgram terminalProgram = new TerminalProgram(userService);
+        ConfigurableApplicationContext context = SpringApplication.run(Runner.class,args);
+        TerminalProgram terminalProgram = context.getBean(TerminalProgram.class);
```

Comment:
you can `@Autowire` `TerminalProgram` to the application. Also you can implement CommandLineRunner to start logic